sub-packages/bionemo-testing/src/bionemo/testing/torch.py

```diff
@@ -0,0 +1,48 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Should we merge this into `utils.py` also?

Reply:
Ugh, I hate the name `utils.py` 😄 . I'd prefer not to but I'm flexible. We're not already importing from there in this test suite, so I think it's OK to split them

Reply:
I am ok with both. `utils.py` sounds too universal.